app/src/main/java/com/teamagam/gimelgimel/app/injectors/modules/ApiModule.java

```diff
@@ -0,0 +1,18 @@
+package com.teamagam.gimelgimel.app.injectors.modules;
+
+import com.teamagam.gimelgimel.data.message.rest.GGMessagingAPI;
+import com.teamagam.gimelgimel.data.message.rest.RestAPI;
+import com.teamagam.gimelgimel.presentation.scopes.PerFragment;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+public class ApiModule {
```

Comment:
1. why it's not part of the repository module?
2, ApiModule is not informative enough.